earth

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+LAST_CHECK_STATE_PATH=/tmp/last-earth-prerelease-check
```

Comment:
We've generally used lower-case variable names for local vars and upper-case for exported vars in bash files. Might be good to maintain style.